ADR/0017-posture-keeper.md

```diff
@@ -0,0 +1,59 @@
+# 17. Posture Keeper
+
+Date: 2023-02-18
+
+## Status
+
+Accepted
+
+## Context
+Posture keeper is a REST-based web service that allows clients to store and retrieve SAST analysis data for a specific git repository and commit SHA. The service is designed to be scalable, reliable, and secure, and uses MongoDB as its underlying data store. The service is implemented using Quarkus. By utilizing this service StoneSoup is going to be able to provides experiences that reflects the current state of security and/or quality posture of the applications.
```

Comment:
I recommend writing more about the background here. What are the multiple problems that we face that drive us to writing the ADR in the first place. Years from now, when we look back and wonder why we created Posture Keeper -- what will we wish we knew about how we were thinking at the time. What are the constraints facing us? (pipeline service workspaces and pvcs get pruned, for example)

Keep mention of the decision (create posture keeper) until the decision section.